gen/builders/generator.go

```diff
@@ -126,37 +169,121 @@ func (g *Generator) MessageVectorGroup(group string, vectors ...*MessageVectorGe
 	go func() {
 		defer g.wg.Done()
 
+		var tmpOutPath string
```

Comment:
Can we call this `tmpOutDir` to make it explicit that this is a path to a directory?

Reply:
We should rename `OutputPath` as well?

Reply:
Yep, could do. OutputDir?